monkey/infection_monkey/credential_collectors/ssh_collector/ssh_handler.py

```diff
@@ -8,13 +8,20 @@
 from infection_monkey.telemetry.attack.t1005_telem import T1005Telem
 from infection_monkey.telemetry.attack.t1145_telem import T1145Telem
 from infection_monkey.telemetry.messengers.i_telemetry_messenger import ITelemetryMessenger
+from infection_monkey.utils.environment import is_windows_os
 
 logger = logging.getLogger(__name__)
 
 DEFAULT_DIRS = ["/.ssh/", "/"]
 
 
 def get_ssh_info(telemetry_messenger: ITelemetryMessenger) -> Iterable[Dict]:
+    if is_windows_os():
+        logger.debug(
+            "Skipping SSH credentials collection because the operating system is not Linux"
+        )
+        return []
```

Comment:
Same thing as with pypykatz. I suppose you added this so that if someone decides to run this collector on windows it wouldn't crash, but I think it should crash.